transcendence-app/nest-cli.json

```diff
@@ -8,7 +8,8 @@
         "name": "@nestjs/swagger",
         "options": {
           "classValidatorShim": true,
-          "introspectComments": true
+          "introspectComments": true,
+          "dtoFileNameSuffix": [".dto.ts", ".entity.ts", ".domain.ts"]
```

Comment:
(I thought I wrote this on Friday, but somehow my review has dissapeared) Anyway, in this case, I was commenting that [dtoFileNameSuffix](https://docs.nestjs.com/openapi/cli-plugin) seems to be assigning, by default, the suffixes that you are explicitly adding here.

> The Swagger plugin will automatically:
> Please, note that your filenames must have one of the following suffixes: ['.dto.ts', '.entity.ts'] (e.g., create-user.dto.ts) in order to be analysed by the plugin.
>
> f you are using a different suffix, you can adjust the plugin's behavior by specifying the dtoFileNameSuffix option (see below).

Are you sure we need to be explicit here?

Reply:
Yes. Once we set some custom values, it overwrites the default ones, and we have to assign them again.